docs/cristin.ttl

```diff
@@ -118,38 +160,46 @@
 
 :arstall_rapportert
     a            rdf:Property ;
-    rdfs:comment "Year that the resource was registered in Cristin." ;
+    rdfs:comment "Year that the resource was included in an NVI report." ;
     rdfs:range   xsd:positiveInteger ;
     rdfs:domain  :Result .
 
 
 :merknadtekst
     a            rdf:Property ;
-    rdfs:comment "UNKNOWN property." ;
+    rdfs:comment "Free text for super-users to add comments when it comes to NVI approvals." ;
     rdfs:range   xsd:string ;
     rdfs:domain  :Result .
 
 :merknadtekst_godkjenning
     a            rdf:Property ;
-    rdfs:comment "UNKNOWN property." ;
+    rdfs:comment "Free text fore super-users to add explanations/details on why the publication was approved." ;
     rdfs:range   xsd:string ;
     rdfs:domain  :Result .
 
 :kildekode
     a            rdf:Property ;
-    rdfs:comment "UNKNOWN property." ;
-    rdfs:range   xsd:anyURI ;
+    rdfs:comment """
+    The source (provider) the result was imported from, if any.
+    Mostly superseded by the list of import references in table VARBEID_KILDE
+    It is a string containing arbitrary codes such as "scopus" "norart" etc.
+    """ ;
+    rdfs:range   xsd:string ;
     rdfs:domain  :Result .
 
 :kildepostid
     a            rdf:Property ;
-    rdfs:comment "UNKNOWN property." ;
-    rdfs:range   xsd:anyURI ;
+    rdfs:comment """
+    The id of the record the result was imported from in the source system (e.g. the SCOPUS id if the result was imported from Scopus).
+     See varbeid.kildekode. Mostly superseded by VARBEID_KILDE. It contains only the identifier found in the source system and cannot be
+     de-referenced.
+    """ ;
+    rdfs:range   xsd:string ;
     rdfs:domain  :Result .
 
 :pubidnr
     a            rdf:Property ;
-    rdfs:comment "UNKNOWN property." ;
+    rdfs:comment "A reference to the record corresponding to this result in the PIA import database" ;
```

Comment:
What is PIA?

Reply:
A prefix/name for automatically imported publications from Elsevier